Makefile

```diff
@@ -29,7 +25,7 @@ clean:
 # remove running and stopped test containers
 	-@ [[ -d config.bak ]] && { rm -rf config ; mv config.bak config ; } || :
 	-@ [[ -d testconfig.bak ]] && { sudo rm -rf test/config ; mv testconfig.bak test/config ; } || :
-	-@ docker ps -a | grep -E "mail|ldap_for_mail|mail_overri.*" | cut -f 1-1 -d ' ' | xargs --no-run-if-empty docker rm -f
+	-@ for container in $$(docker ps -a | grep -E "mail|ldap_for_mail|mail_overri.*|hadolint|eclint|shellcheck" | cut -f 1-1 -d ' '); do docker rm -f $$container; done
```

Comment:
This is problematic/dangerous:

1. `ldap_for_mail|mail_overri.*` will never match, `mail` will match first.
2. All matched containers will be forcefully removed. Think about a container named `webmail` etc.

Reply:
Fixed in https://github.com/docker-mailserver/docker-mailserver/pull/1980/commits/5084caf2da2dd20d7ecc741a8f6bfa7a5952962d